umpost/conversion_driver_esm1p5.py

```diff
@@ -295,25 +296,36 @@ def safe_removal(succeeded, failed):
     return succeeded_inputs - failed_inputs
 
 
-if __name__ == "__main__":
+def parse_args(args):
```

Comment:
I think the `args` arg is superfluous here as `argparse` probably parses `sys.argv` by default.

Reply:
Good point – it works without the extra argument. Have removed this.